File: examples/spaced/ts/commands/elon.ts
/**
 * commands/elon.ts — elon status
 */

import { Command } from 'commander';
import { ELON_QUOTES, pick } from '../data';

const CURRENT_VENTURES = [
  'Tesla (TSLA) — EVs, FSD, energy, shareholder litigation',
  'SpaceX — rockets, Starlink, DOGE-adjacent orbital superiority',
  'X (formerly Twitter) — social media, payments, everything app aspirations',
  'Neuralink — brain-computer interface, FDA approval: partial',
  'The Boring Company — tunnels, Vegas Loop, disrupting nothing yet',
  'xAI (Grok) — AGI, or at least a chatbot with fewer restrictions',
  'DOGE — Department of Government Efficiency (advisory, unofficial, impactful)',
];

const MOOD_POOL = [
  'POSTING (threat level: elevated)',
  'ACQUIRING SOMETHING (due diligence: incomplete)',
  'TWEETING AT REGULATORS (response: pending)',
  'FOUNDING A COMPANY (count: 7)',
  'DISRUPTING (industry: unspecified)',
  'NOMINALIZING AN ANOMALY (outcome: RUD)',
  'OPTIMIZING (headcount: downward)',
];

export function elonCommand(): Command {
  const elon = new Command('elon')
    .description('Elon Musk current status');

  elon
    .command('status')
    .description('Current Elon Musk operational status')
    .action(function () {
      const mood  = pick(MOOD_POOL);
      const quote = pick(ELON_QUOTES);

      console.log('');
      console.log('  ELON MUSK — OPERATIONAL STATUS');
      console.log(`  ${'─'.repeat(50)}`);
      console.log('');
      console.log(`  Current mood:     ${mood}`);
      console.log('');
      console.log('  Active ventures:');
      for (const v of CURRENT_VENTURES) {
        console.log(`    → ${v}`);
      }
      console.log('');
      console.log('  Latest attributed quote:');
      console.log(`    "${quote}"`);
      console.log('');
      console.log('  Net worth:        Varies. Significantly. Check Bloomberg.');
      console.log('  Twitter handle:   @elonmusk (owns the platform; handle irrevocable)');
      console.log('  Government role:  DOGE (unofficial, impactful, conflict-of-interest-ful)');
      console.log('');
      console.log('  Disclaimer: This status is satirical. Elon Musk is a real person');
      console.log('  who has not authorized, reviewed, or endorsed this output.');
      console.log('  He has also not denied it. Make of that what you will.');
      console.log('');
    });

  return elon;
}

File: examples/spaced/py/commands/elon.py
"""commands/elon.py — elon status."""
from __future__ import annotations

import time
import typer

app = typer.Typer(help="Elon Musk current status", add_completion=False)

_QUOTES = [
    "\"The probability that we are not living in a computer simulation is one in billions.\"",
    "\"I would like to die on Mars. Just not on impact.\"",
    "\"When something is important enough, you do it even if the odds are not in your favour.\"",
    "\"Failure is an option here. If things are not failing, you are not innovating enough.\"",
    "\"I think it's very important to have a feedback loop.\" (tweets 47,000 times)",
    "\"We're going to make it. I don't know exactly when, but we're going to make it.\"",
    "\"The first step is to establish that something is possible; then probability will occur.\"",
    "\"Going from PayPal, I thought: 'Well, what are some of the other problems that are likely"
    " to most affect the future of humanity?'\" (answer: rockets, cars, tunnels, AI, brain chips)",
]

_ROLES = [
    "CEO, Tesla",
    "CEO, SpaceX",
    "Owner, X (formerly Twitter)",
    "Founder, xAI",
    "Founder, Neuralink",
    "Founder, The Boring Company",
    "DOGE advisor (self-appointed)",
    "Technoking (official Tesla title)",
]

_STATUS_NOTES = [
    "Currently tweeting. Number of tweets today: unknown (count resets with sanity).",
    "Latest X post: market-moving. SEC: aware.",
    "Currently not sleeping. Reportedly by choice.",
    "In meeting. Meeting is a tweet. Tweet is a policy. Policy is TBD.",
    "Simultaneously running 7 companies. 6 of them are hiring. 1 just laid off 50%.",
]


@app.command("status")
def elon_status() -> None:
    """Display real-time Elon Musk executive status dashboard."""
    idx = int(time.time()) % len(_QUOTES)
    quote = _QUOTES[idx]
    note = _STATUS_NOTES[idx % len(_STATUS_NOTES)]

    roles_str = "\n".join(f"    • {r}" for r in _ROLES)

    typer.echo(f"""
  ELON MUSK — EXECUTIVE STATUS DASHBOARD
  {"─" * 56}
  Full name:    Elon Reeve Musk
  Born:         1971-06-28, Pretoria, South Africa
  Net worth:    fluctuates with TSLA; check Bloomberg Terminal

  Current roles:
{roles_str}

  Quote of the session:
    {quote}

  Activity:
    {note}

  SEC interactions:     ongoing
  Twitter/X posts/day:  classified (high)
  Sleep (hrs/night):    disputed

  Disclaimer: This dashboard is satirical. Any resemblance to actual
  executive behavior is unfortunate but accurate.
""")
